libraries/rush-lib/src/logic/Telemetry.ts

```diff
@@ -66,6 +66,11 @@ export interface ITelemetryOperationResult {
    * If the operation was blocked, will be `undefined`.
    */
   endTimestampMs?: number;
+
+  /**
+   * Duration in seconds when the operation does not hit cache
+   */
+  durationInSecondsWithoutCache?: number;
```

Comment:
originalDurationMs

Reply:
First, the original idea of the property is more like `originalDurationMs`, since it is only specified when the operation hits cache. but, Ian suggests the property should be always specified in "operationResult". So, given these two names a second thought:

1. Let's say `originalDurationMs` wins.

It somewhat weird if it's always be specified. For instance, w/o cache

```
operationResults: {
    "<package_name>": {
        "startTimestampMs": 1300.00,
        "endTimestampMs": 1400.00,
        "originalDurationMs": 100.00,
        "result": "SUCCESS",
        "dependencies": []
    }
}
```

It kind of confuses user why it calls `originalDurationMs` other than `durationMs` 😅

___

2. Let's say `nonCachedDurationMs` wins.

The data in the above case becomes

```
operationResults: {
    "<package_name>": {
        "startTimestampMs": 1300.00,
        "endTimestampMs": 1400.00,
        "nonCachedDurationMs": 100.00,
        "result": "SUCCESS",
        "dependencies": []
    }
}
```

It seems easier for user to understand this property is for `nonCached`, even though the expected `durationMs` is missing.


In conclusion, i would like to choose `nonCachedDurationMs`.